docs/getting-started/plugins.md

```diff
@@ -0,0 +1,148 @@
+# Configure your plugins
+Screwdriver already defaults most configuration, but you can override defaults using a `local.yaml` or environment variables.
+
+## Datastore
+To use DynamoDB, use `dynamodb` plugin. To use Postgres, MySQL, and Sqlite, use `sequelize` plugin.
+
+#### DynamoDB
+Modify your `local.yaml`:
+```yaml
+datastore:
+    plugin: dynamodb
+    dynamodb:
+        # Prefix to the table names
+        prefix: TABLE-PREFIX
+        accessKeyId: AWS-ACCESS-KEY-ID
+        secretAccessKey: AWS-SECRET-ACCESS-KEY
+```
+
+Or set these environment variables in your `docker-compose.yml`:
```

Comment:
There is no `docker-compose.yml` if you don't run ` in-a-box`. Can we make it clear here?